<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>511</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Quit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>481</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Devices</string>
    </property>
    <widget class="QPushButton" name="pushButton_Mode">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_File">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>114</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QComboBox" name="InputDeviceCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="OutputDeviceCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Mode">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>154</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Ouput">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>74</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Ouput</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Input">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>34</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QComboBox" name="FileCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
